CONTRIBUTING.md

```diff
@@ -0,0 +1,136 @@
+- [Testing](#testing)
+- [Code of Conduct](#code-of-conduct)
+- [Getting Help](#getting-help)
+
+
+### Pre Requirements
+
+---
+
+To contribute to our project please download and install the following:
+
+- Vagrant.
+- VirtualBox(or other VM that works with "Vagrant").
+
+After the installations type 'vagrant up' and automatically the development environment will set up.
+
+
+### Issues
+
+---
+
+Issues should be used to report problems with the library, request a new feature, or to discuss potential changes before a PR is created.
+Please make sure that before the PR is created, the issue must be created and after the PR is created, the PR must be linked to the issue relevant to it.
+
+
+### Pull Requests
+
+---
+
+To create a pull request (PR) please follow the instructions:
+
+1. Create an [issue](#Issues).
+2. Fork a repository and clone the repository.
+3. Create a new branch and new remote for the upstream repository.
+4. Make the changes on our local machine, add to the branch, commit the a massage and finally push the PR.
+5. Open a PR in our repository and follow this process: 
+	- Add a detailed description.
+	- Press on the Pull Requests.
+6. Link the issue to the PR when you open issues use there [words](#https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests).
+  Issues can also help the team know what every member of the team work on.
+
+
+### Pull Requests Standards
+
+---
+Our standards for pull request is:
+
+1. Make polite notes to each other without using bad language.
+2. The PRs should be a meaningful and concise title without issue numbers or lines code.
+3. Every PR should be linked to an issue.
+4. Each PR requires the approval of at least two team members.
+
+
+
+### Testing
+
+---
+
+Every PR will be tested by:
+1. flake8 that conforms to proper Python syntax and coding standards.
+2. Automatically checked by a CI system.
+
+
+### Code Of Conduct
+
+---
+
+#### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our 
+community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, 
+gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+
+#### Our Standards
+
+
+Our standards To create a comfortable and positive environment we follow the following rules:
+
+- Using welcoming and inclusive language.
+- Showing empathy towards other community members.
+- Focusing on what is best for the community.
+- Being respectful of differing viewpoints and experiences.
+- Attempting collaboration before conflict.
+- Gracefully accepting constructive criticism.
+
+Our standards To avoid an unacceptable environment we ask not to do the following:
```

Comment:
`Our standards` - does it really needed here?

Reply:
Hi, @svatares I corrected the change according to your suggestion and also rewrote part of the file.
You can see details about the things that were written in commit.